lib/src/rules/use_iterables.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/dart/element/type.dart';
+import 'package:analyzer/dart/element/type_provider.dart';
+
+import '../analyzer.dart';
+
+const _desc = r'Use lazy iterables.';
+
+const _details = r'''
+**DO** use lazy iterables created by `iterable.map`. Since they are lazily
```

Comment:
The word "use" is ambiguous/misleading.

If a style guide/lint tells me to "use" something, I'd think it tells me to *do* put lazy iterables into my code, and that they are not being evaluated is a good thing. Not that I should ensure that the value of said lazy iterable is not ignored.

Q.v. other lints saying things like "Use valid regular expression syntax.".

Maybe "don't ignore the value of" instead of "use"?

Reply:
💯 

Spot on.  Thanks!